files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -14,6 +14,25 @@
 #################
 
 $ModLoad imuxsock # provides support for local system logging
+
+{% if SYSLOG_CONFIG is defined %}
+{% if 'GLOBAL' in SYSLOG_CONFIG %}
+{% if 'rate_limit_interval' in SYSLOG_CONFIG['GLOBAL']%}
+{% set rate_limit_interval = SYSLOG_CONFIG['GLOBAL']['rate_limit_interval'] %}
+{% endif %}
+{% if 'rate_limit_burst' in SYSLOG_CONFIG['GLOBAL']%}
+{% set rate_limit_burst = SYSLOG_CONFIG['GLOBAL']['rate_limit_burst'] %}
+{% endif %}
+{% endif %}
+{% endif %}
+
+{% if rate_limit_interval is defined %}
+$SystemLogRateLimitInterval {{ rate_limit_interval }}
+{% endif %}
+{% if rate_limit_burst is defined %}
+$SystemLogRateLimitBurst {{ rate_limit_burst }}
+{% endif %}
```

Comment:
Same question about else case for providing default values if the feature is not defined.

Reply:
If feature is not defined, there is no configuration of `$SystemLogRateLimitBurst`, `$SystemLogRateLimitInterval`. In that case, there is no rate limiting.